ui/src/authentication/services/AuthenticationService.js

```diff
@@ -1,30 +1,77 @@
+import { http } from '@api';
+import { sessionManager } from './SessionManager';
+
 class AuthenticationService {
-  getTokenFromDOM = () => {
-    const element = document.getElementsByName('authorized_token');
-    if (element.length > 0 && element[0].hasAttribute('value')) {
-      return element[0].value;
+  loginWithOauthProvider = (nextUrl, providerUrl) => {
+    localStorage.setItem('ILS_USER_WAS_LOGGEDIN', true);
+    sessionManager.setAnonymous();
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
why this `if`?

Reply:
Because on development the hostnames are different but in production should keep the window hostname...